.travis.yml

```diff
@@ -0,0 +1,37 @@
+language: python
+
+os:
+  - linux
+  - osx
+
+branches:
+  only:
+    - master
+
+git:
+  depth: 10
+
+
+matrix:
+  include:
+    - os: linux
+      env:
+        - ATOM_CHANNEL=stable
+        - ATOM_CHANNEL=beta
```

Comment:
Unfortunately since we are in a matrix it doesn't do the auto-expansion in here, you'll need a separate entry for each of these.